src/LessMsi.Gui/Model/StreamInfoView.cs

```diff
@@ -47,11 +49,16 @@ public string Name
 			get { return _name; }
 		}
 
+		public string DisplayName
+		{
+			get { return _displayName; }
+		}
+
 		public bool IsCabStream { get; private set; }
 
 		public string Label
 		{
-			get { return this.IsCabStream ? this.Name + " (CAB)" : this.Name; }
+			get { return this.IsCabStream ? this.DisplayName + " (CAB)" : this.DisplayName; }
```

Comment:
@activescott I intentionally only used it here for displaying, since the 'Name' is also used for other things like extracting.

Reply:
Great context. Thanks for the comment!